Elemental/Components/AePagination/AePagination.razor

```diff
@@ -1,98 +1,120 @@
+	[Parameter] public IPageableComponent Pageable { get; set; }
+	[Parameter] public EventCallback<int> OnPageClick { get; set; }
+
+	[Parameter] public RenderFragment LeftIcon { get; set; }
+	[Parameter] public RenderFragment RightIcon { get; set; }
+	
+
+	private int _numberOfButtonsOnSide = 2;
+	private int _numberOfButtons => _numberOfButtonsOnSide * 2 + 1;
+	private int _middleButton => Math.Min((Pageable.PageCount - 1) - (_numberOfButtonsOnSide + 1),
+												Math.Max(Pageable.Page, _numberOfButtonsOnSide + 1));
+	private int _abbreviateMin => Math.Max(1, Pageable.Page - _numberOfButtonsOnSide);
+	private int _abbreviateMax => Math.Min(Pageable.PageCount - 1, Pageable.Page + _numberOfButtonsOnSide);
+
 
-  private int _numberOfButtonsOnSide = 2;
-  private int _numberOfButtons => _numberOfButtonsOnSide * 2 + 1;
-  private int _middleButton => Math.Min((Pageable.PageCount-1) - (_numberOfButtonsOnSide+1), 
-                                        Math.Max(Pageable.Page, _numberOfButtonsOnSide+1));
-  private int _abbreviateMin => Math.Max(1, Pageable.Page - _numberOfButtonsOnSide);
-  private int _abbreviateMax => Math.Min(Pageable.PageCount - 1, Pageable.Page + _numberOfButtonsOnSide);
-  
+	private string _className => $"ae pagination {_inputClass}";
+	private string _pageLeftClass => $"ae page page-left {(Pageable.Page == 0 ? "disabled" : "")}";
+	private string _pageRightClass => $"ae page page-right {(Pageable.Page == Pageable.PageCount - 1 ? "disabled" : "")}";
 
-  private string _className => $"ae pagination {_inputClass}";
-  private string _pageLeftClass => $"ae page page-left {(Pageable.Page == 0 ? "disabled" : "")}";
-  private string _pageRightClass => $"ae page page-right {(Pageable.Page == Pageable.PageCount - 1 ? "disabled" : "")}";
+	private string GetPageClass(int localIndex, bool nuzzled)
+	{
+		var active = localIndex == Pageable.Page ? "active" : "";
+		var disabled = nuzzled ? "disabled" : "";
+		return $"ae page {active} {disabled}";
+	}
+	private async Task PageClicked(int targetIndex, bool nuzzled = false)
+	{
+		if (nuzzled || targetIndex < 0 || targetIndex >= Pageable.PageCount || targetIndex == Pageable.Page)
+		{
+			return;
+		}
 
-  private string GetPageClass(int localIndex, bool nuzzled)
-  {
-    var active = localIndex == Pageable.Page ? "active" : "";
-    var disabled = nuzzled ? "disabled" : "";
-    return $"ae page {active} {disabled}";
-  }
-  private async Task PageClicked(int targetIndex, bool nuzzled = false)
-  {
-    if(nuzzled || targetIndex < 0 || targetIndex >= Pageable.PageCount || targetIndex == Pageable.Page){
-      return;
-    }
+		await OnPageClick.InvokeAsync(targetIndex);
+	}
 
-    await OnPageClick.InvokeAsync(targetIndex);
-  }
+	private string _wrapperClassname => $"ae pagination-wrapper {_inputClass}";
+    private string _wrappedInputClassname => $"ae input {(LeftIcon != null ? "has-left" : "")} {(RightIcon != null ? "has-right" : "")}";
```

Comment:
I can't find these variables - are they used?

Reply:
I don't think they are used, but they were there before I made changes, so I just left it be since I wasn't sure if it was being used.

Reply:
Gotcha, I think it's a relic left over then - let's clean them up